### PR TITLE
glslang: new port (submission)

### PR DESCRIPTION
**Pull request by eintw1ck**
###### Tested on
macOS 10.14.4 18E226
Xcode 10.2 10E125

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] checked your Portfile with `port lint`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: graphics/glslang/Portfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        KhronosGroup glslang 7.11.3113
+
+categories          graphics
+platforms           darwin
+license             {BSD Permissive}
+
+maintainers         {itanimul.li:josh @eintw1ck}
```
Comment (cjones051073) [inline graphics/glslang/Portfile]: Please add openmaintainer here.

File: graphics/glslang/Portfile [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+description         Shader compiler libraries & tools
+long_description    OpenGL and OpenGL ES reference compiler for shading languages
+
+homepage            https://www.khronos.org/opengles/sdk/tools/Reference-Compiler/
+checksums           sha256 18f6866f1607b842abfa461dd6e8728f6fa173d1fa95a5cdc625afa10c878506
+use_zip             no
+
+PortGroup           cmake 1.0
```
Comment (cjones051073) [inline graphics/glslang/Portfile]: Conventionally we place all portgroups at the top, so please move this up.

File: graphics/glslang/Portfile [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+categories          graphics
+platforms           darwin
+license             {BSD Permissive}
+
+maintainers         {itanimul.li:josh @eintw1ck}
+description         Shader compiler libraries & tools
+long_description    OpenGL and OpenGL ES reference compiler for shading languages
+
+homepage            https://www.khronos.org/opengles/sdk/tools/Reference-Compiler/
+checksums           sha256 18f6866f1607b842abfa461dd6e8728f6fa173d1fa95a5cdc625afa10c878506
```
Comment (cjones051073) [inline graphics/glslang/Portfile]: Please add rmd160 and size checksums here

**Comment by eintw1ck** [comment]
@cjones051073 thanks for review, I have fixed the issues you mentioned.

**Comment by cjones051073** [comment]
thanks, looks good now. will let checks complete though before merging.

**Pull request merged**